crates/extensions/c8y_mapper_ext/src/config.rs

```diff
@@ -141,10 +141,9 @@ impl C8yMapperConfig {
     pub fn default_external_topic_filter() -> TopicFilter {
         vec![
             "te/+/+/+/+/m/+",
+            "te/+/+/+/+/e/+",
             "tedge/alarms/+/+",
             "tedge/alarms/+/+/+",
-            "tedge/events/+",
-            "tedge/events/+/+",
```

Comment:
Seeing these `tedge/` topics replaced by the `te/` topics, while the former is only deprecated for the next release but not removed, I feel I'm missing something here. Especially since the same was done for `tedge/measurements` as well.

Reply:
The agent does the translation from `tedge/#` to `te/#`.

This has been implemented by https://github.com/thin-edge/thin-edge.io/pull/2125